lib/node_modules/@stdlib/blas/ext/index-of/lib/main.js

```diff
@@ -0,0 +1,204 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var hasOwnProp = require( '@stdlib/assert/has-own-property' );
+var isNumber = require( '@stdlib/assert/is-number' ).isPrimitive;
+var isndarrayLike = require( '@stdlib/assert/is-ndarray-like' );
+var broadcastScalar = require( '@stdlib/ndarray/base/broadcast-scalar' );
+var maybeBroadcastArray = require( '@stdlib/ndarray/base/maybe-broadcast-array' );
+var getDType = require( '@stdlib/ndarray/dtype' );
+var getShape = require( '@stdlib/ndarray/shape' );
+var getOrder = require( '@stdlib/ndarray/order' );
+var format = require( '@stdlib/string/format' );
+var defaults = require( '@stdlib/ndarray/defaults' );
+var nonCoreShape = require( './non_core_shape.js' );
+var base = require( './base.js' );
+
+
+// VARIABLES //
+
+var DEFAULT_DTYPE = defaults.get( 'dtypes.integer_index' );
+
+
+// MAIN //
+
+/**
+* Returns the first index of a specified search element along one or more ndarray dimensions.
+*
+* @name indexOf
+* @type {Function}
+* @param {ndarrayLike} x - input ndarray
+* @param {(ndarrayLike|number)} searchElement - search element
```

Comment:
```suggestion
* @param {(ndarrayLike|*)} searchElement - search element
```

Reply:
Please fix this here and everywhere else. Having to correct this is getting repetitive.